Loop/Managers/OnboardingManager.swift

```diff
@@ -92,7 +92,12 @@ class OnboardingManager {
             return
         }
 
-        displayOnboarding(onboarding)
+        let resuming = isSuspended
```

Comment:
what is the difference between `resumeSuspended` and `resuming`?

Reply:
The incoming parameter `resumeSuspended` means "if onboarding is currently suspended, then it is okay to resume it; otherwise do not resume and continue launch to home screen as normal. The local variable `resuming` is simply a flag indicating whether the onboarding is actually resuming or not (it was suspended and it is allowed to resume).

I suppose I could rename `resumeSuspended` to `allowResume` or something like that. Will that help, @nhamming?

Reply:
Done.